nautobot/extras/health_checks.py

```diff
@@ -67,6 +102,12 @@ def check_redis(self, redis_url=None, **kwargs):
 class CacheopsRedisBackend(RedisHealthCheck):
     """Health check for Cacheops Redis."""
 
+    metric = Gauge(
+        "health_check_cache_ops_redis_info",
+        "State of cache ops redis backend",
+        multiprocess_mode=NautobotHealthCheckBackend.MULTIPROCESS_MODE,
+    )
+
```

Comment:
Per your comment this doesn't seem to be working and CacheOps is going away soon. YAGNI.